sample-compose/app/src/main/java/co/nimblehq/sample/compose/model/UiModel.kt

```diff
@@ -6,7 +6,7 @@ import kotlinx.parcelize.Parcelize
 
 @Parcelize
 data class UiModel(
-    val id: String
+    val id: String = ""
```

Comment:
Default value is no longer needed 
```suggestion
    val id: String
```